examples/audio_control.rs

```diff
@@ -0,0 +1,76 @@
+//! This example illustrates how to load and play an audio file.
+//! For loading additional audio formats, you can enable the corresponding feature for that audio format.
+
+use bevy::prelude::*;
+use bevy::utils::error;
+use bevy_kira_components::kira::sound::{PlaybackRate, PlaybackState};
+use bevy_kira_components::kira::tween::Tween;
+use bevy_kira_components::prelude::AudioBundle;
+use bevy_kira_components::prelude::*;
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins)
+        .add_plugins(AudioPlugin)
+        .add_systems(Startup, setup)
+        .add_systems(Update, (update_speed, pause, volume))
+        .run();
+}
+
+fn setup(asset_server: Res<AssetServer>, mut commands: Commands) {
+    commands.spawn((
+        AudioBundle {
+            source: asset_server.load::<AudioFile>("Windless Slopes.ogg"),
```

Comment:
Actually, missed that, you can use `AudioFileBundle` which is a simple type alias, but sets the type of `source` so that you don't have to specify it (similarly to how `PbrMaterialBundle` works)

```suggestion
        AudioFileBundle {
            source: asset_server.load("Windless Slopes.ogg"),
```